### PR TITLE
Upload: --quiet option for hca upload file

### PR DESCRIPTION
**Pull request by sampierson**
and make default action to report progress.

### CODE REVIEW

**Comment by sampierson** [comment]
Silly GitHub closed this PR because it was stacked.  Reopening.

File: hca/upload/cli/upload_command.py [hunk 1]
```diff
@@ -20,19 +20,22 @@ def add_parser(cls, upload_subparsers):
...
                                    " Only valid when one file is being uploaded.")
+        upload_parser.add_argument('-q', '--quiet', action='store_true', help="Suppress normal output.")
         upload_parser.set_defaults(func=UploadCommand)
 
     def __init__(self, args):
         self._load_config()
         self._check_args(args)
         for file_path in args.file_paths:
-            self._upload_file(file_path, args.target_filename)
+            self._upload_file(file_path, args.target_filename, report_progress=(not args.quiet))
```
Comment (ttung) [inline hca/upload/cli/upload_command.py]: I would just go with quiet to make it consistent with the rest of the codebase.

Comment (sampierson) [reply hca/upload/cli/upload_command.py]: You mean the name of the kwarg?

Comment (ttung) [reply hca/upload/cli/upload_command.py]: yeah.

Comment (sampierson) [reply hca/upload/cli/upload_command.py]: I'm coding it up, bit it feels unintuitive having the default behavior of the python binding be to generate output on stdout, or to have the default value of the quiet kwarg be True.  So I'm going to leave it as is.

**Pull request merged**